src/lib.rs

```diff
@@ -192,10 +186,50 @@ pub trait UnicodeSegmentation {
     /// [Alphabetic](http://unicode.org/reports/tr44/#Alphabetic)
     /// property, or with
     /// [General_Category=Number](http://unicode.org/reports/tr44/#General_Category_Values).
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use self::unicode_segmentation::UnicodeSegmentation;
+    /// let uss = "Mr. Fox jumped. [...] The dog was too lazy.";
+    /// let us1 = uss.unicode_sentences().collect::<Vec<&str>>();
+    /// let b: &[_] = &["Mr. ", "Fox jumped. ", "The dog was too lazy."];
+    ///
+    /// assert_eq!(&us1[..], b);
+    /// ```
+    fn unicode_sentences<'a>(&'a self) -> UnicodeSentences<'a>;
```

Comment:
Can we add some text explaining what this function does?

Reply:
I'm afraid I don't understand. Do you mean I should expand on

> Returns an iterator over substrings of `self` separated on
> [UAX#29 sentence boundaries](http://www.unicode.org/reports/tr29/#Sentence_Boundaries).
>
> Here, "sentences" are just those substrings which, after splitting on
> UAX#29 sentence boundaries, contain any alphanumeric characters. That is, the
> substring must contain at least one character with the
> [Alphabetic](http://unicode.org/reports/tr44/#Alphabetic)
> property, or with
> [General_Category=Number](http://unicode.org/reports/tr44/#General_Category_Values).

above the example?

(I'm wondering whether the way the diff is rendered might be confusing you.)

Reply:
oh, sorry, this was absolutely diff rendering stuff 😄